heat/core/tests/test_manipulations.py

```diff
@@ -874,6 +874,49 @@ def test_hstack(self):
         res = ht.hstack((a, b))
         self.assertEqual(res.shape, (24,))
 
+    def test_reshape(self):
+        # split = None
+        a = ht.zeros((3, 4), device=ht_device)
+        result = ht.zeros((2, 6), device=ht_device)
+        reshaped = ht.reshape(a, (2, 6))
+
+        self.assertEqual(reshaped.size, result.size)
+        self.assertEqual(reshaped.shape, result.shape)
+        self.assertTrue(ht.equal(reshaped, result))
+
+        # 1-dim distributed vector
+        a = ht.arange(8, dtype=ht.float32, split=0)
+        result = ht.array([[[0, 1], [2, 3]], [[4, 5], [6, 7]]], dtype=ht.float32, split=0)
+        reshaped = ht.reshape(a, (2, 2, 2))
+
+        self.assertEqual(reshaped.size, result.size)
+        self.assertEqual(reshaped.shape, result.shape)
+        self.assertTrue(ht.equal(reshaped, result))
```

Comment:
test for dtype

Reply:
also we use floats as the default so this doesnt test anything different. please change to float64